reverse_engineering/helpers/getFieldAttributes.js

```diff
@@ -9,18 +9,18 @@ let _;
 const getFieldAttributes = ({ attributes = {}, type = '' }) => {
 	_ = dependencies.lodash;
 
-	return _.flow([
-	    filterAttributes(type),
-	    setNamespace(type),
-	    setSubtype(type),
-	    setDescriptionFromDoc,
-	    convertDefaultValue,
-	    setDurationSize,
-	    addMetaProperties,
-	])(attributes);
+	let fieldAttributes = filterAttributes(attributes, type);
```

Comment:
just get rid from flow - debugging of this is painfull for now. And as I reordered arguments in filterAttributes - it's not so appliable for _.flow now